5-Access-Authentication-and-Authorization/5.3-Configure-SSH-Server.bats

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env bats
+
+@test "5.3.1 Ensure permissions on /etc/ssh/sshd_config are configured (Automated)" {
+    run bash -c "stat /etc/ssh/sshd_config"
+    [ "$status" -eq 0 ]
+    [[ "$output" = *"Uid:"*"("*"0/"*"root"* ]]
+    [[ "$output" = *"Gid:"*"("*"0/"*"root"* ]]
+    [[ "$output" = *"Access:"*"(0"[0-7]"00"* ]]
+}
+
+@test "5.3.2 Ensure permissions on SSH private host key files are configured (Automated)" {
+    local SSH_CONFIG=$(find /etc/ssh -xdev -type f -name 'ssh_host_*_key' -exec stat {} \; | grep "Access" | grep "Uid")
+    while IFS= read -r line; do
+        [[ "$line" = *"Uid:"*"("*"0/"*"root"* ]]
+        [[ "$line" = *"Gid:"*"("*"0/"*"root"* ]]
+        [[ "$line" = "Access:"*"(0"[0-7]"00"* ]]
```

Comment:
According to the CIS document permissions have to be "0600 or more restrictive"
```suggestion
        [[ "$line" = "Access:"*"(0"[0-6]"00"* ]]
```

Reply:
According to the document `group` and `other` should not have any permissions. `600` is the default value and in this case just an example output. So `700` is totally fine in this place.

Reply:
That's what they say in 5.3.1, but in 5.3.2 the document explicitly says `... verify Uid is 0/root and Gid is 0/root and permissions are 0600 or more restrictive`